cicd/base/config.ini

```diff
@@ -0,0 +1,36 @@
+[DEFAULT]
+PRODUCT_COMPONENT = base
+PIPELINE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-pipeline-${ENVIRONMENT}
+BASE_STACK_FILE_PFIX = base
+BASE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-${ENVIRONMENT}
+AUTO_DEPLOY = false
+SLACK_DEPLOY_NOTIFICATION = false
+SLACK_CHANNEL = deployments
+DEFAULT_LOGOUT_REDIRECT_URL = https://www.ala.org.au
+
+[development]
+# code pipeline
+PIPELINE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-pipeline-${CLEAN_BRANCH}
+BASE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-${CLEAN_BRANCH}
+SLACK_DEPLOY_NOTIFICATION = true
+SLACK_CHANNEL = zabbix-userdetails
+AUTO_DEPLOY = true
+
+COGNITO_STACK_NAME = ala-cognito-pool-testing
+APP_CLIENT_DOMAIN = ${PRODUCT_NAME}-${CLEAN_BRANCH}.${HOSTED_ZONE}
+DEFAULT_LOGOUT_REDIRECT_URL = https://www-test.ala.org.au
+
+[testing]
+PIPELINE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-pipeline-${CLEAN_BRANCH}
+BASE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-${CLEAN_BRANCH}
```

Comment:
PIPELINE_STACK_NAME and BASE_STACK_NAME can be removed from the [testing] config, then the stack names will use the default:

PIPELINE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-pipeline-${ENVIRONMENT}
BASE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-${ENVIRONMENT}